bin/quality

```diff
@@ -0,0 +1 @@
+bash lint && bash coverage && echo "All good!"
```

Comment:
This script does not work using ```./bin/quality``` since these two other scripts are not in the current directory.